src/DynamoDb/SessionConnectionConfigTrait.php

```diff
@@ -0,0 +1,244 @@
+<?php
+namespace Aws\DynamoDb;
+
+trait SessionConnectionConfigTrait
+{
+    /** @var string Name of table to store the sessions */
+    protected $table_name = 'sessions';
```

Comment:
All of these variables should be camelCased.

Reply:
@srchase regarding the tests - totally agree with you.
I actually covered both standard and custom tests here:
https://github.com/alessionobile/aws-sdk-php/blob/master/tests/DynamoDb/StandardSessionConnectionTest.php#L18
and here:
https://github.com/alessionobile/aws-sdk-php/blob/master/tests/DynamoDb/StandardSessionConnectionTest.php#L34

But perhaps if you think is best i can create a specific test case for it

Reply:
@srchase added a specific test for the ConfigTrait.
https://github.com/alessionobile/aws-sdk-php/blob/master/tests/DynamoDb/SessionConnectionConfigTraitTest.php

I kept the standard/custom config test anyway within the StandardConnection test case. What do you think? :) Better to keep it there or take it away and rely only in SessionConnectionConfigTraitTest.php?